docs/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md

```diff
@@ -0,0 +1,14 @@
+---
+id: platforms
+title: "Camunda Platform 8 officially supported Kubernetes platforms"
+description: "An overview of Camunda Platform 8 officially supported Kubernetes platforms"
+---
+
+import DocCardList from '@theme/DocCardList';
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
+
+[Deploying Camunda Platform 8 using Helm charts](../deploy.md) will work for all platforms and distributions. However, each platform or cloud provider may have some special prerequisites or pitfalls to avoid. The platforms guides highlights important notes for a smooth Camunda Platform deployment on various platforms.
```

Comment:
"The platforms guides highlights important notes for a smooth Camunda Platform deployment on various platforms."

Do you mean "The Platform guides highlight important notes..."? Could we link to what you've mentioned here?

Reply:
@christinaausley I've rephrased it. I was referencing to "Kubernetes platform" not the "Camunda Platform".

Regarding the links, they are auto-generated at the end of the page, as shown in this screenshot:
![201646460-94ee85f6-23e2-49f1-b999-95a21aad5289](https://user-images.githubusercontent.com/6760103/202098738-3fdc99b1-e90a-4367-afcf-aaaa84852ce4.png)